dd-sdk-android-rx/apiSurface

```diff
@@ -0,0 +1,8 @@
+class com.datadog.android.rx.DatadogErrorConsumer : io.reactivex.rxjava3.functions.Consumer<Throwable>
+  override fun accept(Throwable)
+  companion object 
+fun <T> publishErrorsToRum(): io.reactivex.rxjava3.core.Observable<T>
```

Comment:
instead of publish (which might create confusion with the `Publisher` API in Rx), maybe we could name those methods `sendErrorToDatadog`. You can also drop the `s` in `Errors` because a stream can't have more than one Error IIRC.